test/qa/lib/settings.py

```diff
@@ -51,3 +51,52 @@ def get_is_notify_enabled():
 def get_is_routing_enabled():
     """Returns True, if Routing is enabled in application settings."""
     return "Routing: enabled" in sh.nordvpn.settings()
+
+
+def get_is_autoconnect_enabled():
```

Comment:
I saw that also other functions are named like this before, but if the function returns true/false it might be better if it starts with `is`/`has`? e.g.: `is_autoconnect_enabled`?
also for the others.

Reply:
Yes, I agree that this would be better, will change for previous settings as well